aspnet/getting-started/installing-on-windows.rst

```diff
@@ -21,21 +26,18 @@ Once Visual Studio is installed, ASP.NET 5 is installed as well. You're ready to
 Install ASP.NET Standalone
 --------------------------
 
-Visual Studio isn't the only way to install ASP.NET, and installing an IDE may not be appropriate in some scenarios. You can also install ASP.NET on its own from a command prompt. There are a few steps involved, since we'll need to install and configure the environment in which ASP.NET runs, known as the .NET Execution Environment (DNX). Before installing DNX, we need one more tool, the .NET Version Manager (DNVM).
+Visual Studio isn't the only way to install ASP.NET, and installing an IDE may not be appropriate in some scenarios. You can also install ASP.NET on its own from a command prompt. There are a few steps involved, since we'll need to install and configure the environment in which ASP.NET runs, known as the .NET Execution Environment (DNX). To install DNX, we need one more tool, the .NET Version Manager (DNVM).
 
 Install the .NET Version Manager (DNVM)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The .NET Version Manager is used to install one or more versions of the .NET Execution Environment, and to manage which version is currently active. To install DNVM on Windows, you need to open a command prompt as an Administrator, and run the following Powershell script:
-
-.. code-block:: console
-
+The .NET Version Manager is used to install one or more versions of the .NET Execution Environment (DNX), and to manage which version is currently active. 
 
-	@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}"
+To install DNVM on Windows, download and run the appropriate installer for your operating system type:
+- `.NET Version Manager (x86) <http://download.microsoft.com/download/2/4/F/24FABE35-76C4-42DD-AA36-0909B18AD74B/DotNetVersionManager-x86.msi>`__
+- `.NET Version Manager (x64) <http://download.microsoft.com/download/2/4/F/24FABE35-76C4-42DD-AA36-0909B18AD74B/DotNetVersionManager-x64.msi>`__
```

Comment:
Do you want to mention most dev's will select x64. You can't run the wrong installer, so if you select the wrong version you'll be notified.